recipes/rvlib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "rvlib" %}
+{% set version = "0.0.1" %}
+{% set sha1 = "40110f5cf76a0fa29fa979179d66126170bf4d58" %} 
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantEcon/rvlib/archive/{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+
+build:
+  skip: True  # [win or not py35]
+  number: 2
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python  # [py35]
+    - setuptools
+    - numpy
+    - cffi
+    - numba
+    - pyyaml
+
+  run:
+    - python  # [py35]
+    - numpy
+    - cffi
+    - numba
+    - pyyaml
+
+
+test:
+  imports:
+    - rvlib
+
+about:
+  home: https://github.com/QuantEcon/rvlib.git
+  license: BSD-3
```

Comment:
So we are ok with people working from development branches and the like while they are getting started. We just like to iron it out as a release in the end before it it leaves staged-recipes. This will avoid the need to move tags around. If that workflow would be helpful for you in the future, maybe that would be a good way to go.